app/views/guidance/_database_only.html.erb

```diff
@@ -0,0 +1,62 @@
+<p class="govuk-body">
+  Use the table to check how data maps between HESA and Register trainee teachers (Register).
+</p>
+
+<p class="govuk-body">
+  Where possible, these tables follow the same order as the CSV export and a trainee’s record in Register.
+</p>
```

Comment:
These paragraphs are duplicates of the main page ones - the prototype has different content.